Classes/Core/Functional/Framework/DataHandling/Scenario/DataHandlerFactory.php

```diff
@@ -158,8 +161,22 @@ private function processEntityItem(
             $this->processEntities(
                 $itemSettings['entities'],
                 $newId,
-                $parentId
+                $parentId,
+                $entityConfiguration,
+            );
+        }
+        if (
+            $parentEntityConfiguration instanceof EntityConfiguration &&
+            $entityConfiguration->getParentRelationColumnName() !== null &&
+            $entityConfiguration->getParentRelationColumnName() !== '') {
+            $existingValues = array_filter(
+                explode(
+                    ',',
+                    (string)($this->dataMapPerWorkspace[$workspaceId][$parentEntityConfiguration->getTableName()][$nodeId][$entityConfiguration->getParentRelationColumnName()] ?? '')
+                )
             );
+            $existingValues[] = $newId;
+            $this->dataMapPerWorkspace[$workspaceId][$parentEntityConfiguration->getTableName()][$nodeId][$entityConfiguration->getParentRelationColumnName()] = implode(',', $existingValues);
```

Comment:
This implementation would only work for entities that are defined as “nodes”. It would therefore only work for pages.